exercises/atbash-cipher/description.md

```diff
@@ -15,10 +15,11 @@ a simple monoalphabetic substitution cipher. However, this may not have
 been an issue in the cipher's time.
 
 Ciphertext is written out in groups of fixed length, the traditional group size
-being 5 letters, and punctuation is excluded. This is to make it harder to guess
-things based on word boundaries.
+being 5 letters, and punctuation is excluded. This is to make it harder to
+guess things based on word boundaries.
 
 ## Examples
 - Encoding `test` gives `gvhg`
 - Decoding `gvhg` gives `test`
-- Decoding `gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt` gives `The quick brown fox jumps over the lazy dog.`
+- Decoding `gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt` gives `The quick brown
+  fox jumps over the lazy dog.`
```

Comment:
Agreed. In the cases where we have literal text I think we should let them overflow the defined column width.